tests/integration/targets/apache2_module/tasks/635-apache2-misleading-warning.yml

```diff
@@ -0,0 +1,34 @@
+---
+# This test represent the misleading behavior of the following issue: https://github.com/ansible-collections/community.general/issues/635
+- name: Disable MPM event module
+  apache2_module:
+    name: "{{ item.module}}"
+    state: "{{ item.state}}"
+    ignore_configcheck: true
+  register: disable_mpm_modules
+  with_items:
+    - { module: mpm_event, state: absent }
+    - { module: mpm_prefork, state: present }
+
+- assert:
+    that:
+    - "'warnings' in disable_mpm_modules"
+    - disable_mpm_modules["warnings"] == [
+      "No MPM module loaded! apache2 reload AND other module actions will fail if no MPM module is loaded immediately.",
+      "No MPM module loaded! apache2 reload AND other module actions will fail if no MPM module is loaded immediately."
+      ]
+
+- name: Disable MPM event module
+  apache2_module:
+    name: "{{ item.module}}"
+    state: "{{ item.state}}"
+    ignore_configcheck: true
+    warn_mpm_absent: False
+  register: disable_mpm_modules
+  with_items:
+    - { module: mpm_event, state: absent }
+    - { module: mpm_prefork, state: present }
```

Comment:
Since this invocation does the same as the previous one, and thus shouldn't make any change, shouldn't it also not print any warning *without* `warn_mpm_absent=false`?

Reply:
I wonder... :)

I'm going to refine that test.

Reply:
Indeed, that test was wrong. I've added a statement to revert the situation and do an effective test.